test/perf/heap-32-bench.mo

```diff
@@ -0,0 +1,48 @@
+// allocate 2 big arrays with a 10^5 entries each, and populate them
+// with Int32s and Nat32s (somewhat randomly)
+import Prim "mo:⛔";
```

Comment:
I'll break out this benchmark to another PR, so it gets  merged ahead of this one. Then we'll see the diffs for more.

Reply:
Still, comments are warmly welcomed!

Reply:
I see, I didn't explicitly register that. Would you suggest a new category (`test/benchmarks`) or scrapping this test altogether?

Reply:
Separate microbenchmarks would be fine. What I want to avoid is that we think some random change had a significant real world perf impact, one way or another, when it “just” triggered a microbenchmark.

As for more reporting: the data is there, the report can be extended to contain more data. Maybe hidden in `<details>`?

Reply:
I would also spare the user from seeing a comment at all coming from benchmarks, unless there is an actual cycle/size change in one of them.